scripts/e2e-tests.sh

```diff
@@ -9,4 +9,4 @@ cargo build --release --target wasm32-unknown-unknown -Z build-std=std,panic_abo
 
 export RPC_URL=http://localhost:8547
 
-cargo test --features std,e2e --test "*"
+cargo test --features std,e2e --test "erc1155-supply"
```

Comment:
```suggestion
cargo test --features std,e2e --test "*"
```

Reply:
We could have an input argument for this script with a default `*` value, like: `${1:-*}`, so we can avoid the need to change this script for local testing